---
title: "Choi Lab - Research"
layout: textlay
excerpt: "Choi Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

### The genomic architecture of adaptive radiation

<table class="image" align="left" >
<caption style="margin-top: -150px" ><i>Metrosideros polymorpha</i> var. incana with Mauna Loa in the background.<br>Photo taken by JY Choi.</caption>
<td><tr><img src = "{{ site.url}}{{ site.baseurl}}/images/ResearchPage/MpolInc.jpeg" class="img-responsive" align="left" width = "45%" style="border-radius:0%; margin-bottom: 150px" ></tr></td>
</table>
Adaptive radiation—the evolutionary phenomenon where a single ancestral lineage diversifies into a variety of adaptive forms—has fascinated evolutionary biologists since Darwin. These extraordinary morphological and ecological diversifications have long captivated biologists, and example radiations have been model systems to study the link between ecology and speciation. Adaptive radiations in remote settings such as islands are especially intriguing and even paradoxical. How can rapid species diversification occur from a genetically bottlenecked founder population?

Our research addresses this question by studying the Hawaiian Metrosideros and understand the evolutionary basis of island adaptive radiation in plants. The Metrosideros group is one of the most widely dispersed flowering plants in the Pacific Basin, growing from the Philippines to Chile. The Hawaiian archipelago is particularly rich with 23 morphotypes that are non-randomly distributed across the island. Through collaboration with [Elizabeth Stacy (UNLV)](https://estacy.faculty.unlv.edu/){:target="_blank"}, our research implements evolutionary and population genomic approaches to study the biological basis of the Metrosideros adaptive radiation

Related work: [Choi et al. 2021. PNAS](https://pubmed.ncbi.nlm.nih.gov/34497122/){:target="_blank"}; [Choi et al. 2020. Mol Biol Evol](https://pubmed.ncbi.nlm.nih.gov/31693149/){:target="_blank"}<br>
Evolution meeting 2022 talk on Hawaiian Metrosideros genomics: [link](https://www.youtube.com/watch?v=5GdiXSPMuBo&ab_channel=ASNAmNat){:target="_blank"}
<br><br>

### Telomere variation and life history evolution

<table class="image" align="left" >
<caption style="margin-top: -100px" >Coevolution within the TERT-TR complex is a major interest of our group.</caption>
<td><tr><img src = "{{ site.url}}{{ site.baseurl}}/images/ResearchPage/TertTR.jpeg" class="img-responsive" width = "45%" style="border-radius:0%; float: left; margin-bottom: 100px" /></tr></td>
</table>

Structural variations (SVs) are large complicated sequence changes across the genome. Its evolutionary significance is largely elusive due to the difficulties of its detection. Our research focuses on detecting and studying the evolutionary consequence of SVs, by applying state-of-the-art genome sequencing technology and collecting phenotypic data from the field and the lab.

One particular SV within the chromosome are those found in the telomere. Telomeres maintain chromosome ends from damage and are under functional constraint as aberrations can have dire consequences. Paradoxically, telomere lengths vary naturally between individuals, for instance telomere lengths can vary over 10-fold in Arabidopsis thaliana. But why this variability is observed in plants is not known. 

Our previous research has discovered plant telomeres are a target of selection and confers adaptive benefits in specific life history or ecological conditions. Importantly the study suggests telomeres are novel genetic source of adaptive traits in plants, emphasizing chromosome structures as understudied genetic variations of plant ecological diversification. Our research aims to understand the molecular link between flowering and telomere structure in plants, to understand the biological basis of chromosome structure and life history trait evolution.
  
Related work: [Kumawat et al. 2023. bioRxiv](https://www.biorxiv.org/content/10.1101/2023.12.06.568249v1){:target="_blank"}; [Choi et al. 2021. The Plant Cell](https://pubmed.ncbi.nlm.nih.gov/33580702/){:target="_blank"}
TAGC 2024 talk on Mimulus telomere evolution: [link](https://youtu.be/rPXR_-_w3mM){:target="_blank"}
<br><br>

### Developing and applying genomic techniques for evolutionary research

<table class="image" align="left" >
<caption style="margin-top: -150px" >Diagram of PoreC-seq.</caption>
<td><tr><img src = "{{ site.url}}{{ site.baseurl}}/images/ResearchPage/Tech.jpeg" class="img-responsive" width = "45%" style="border-radius:0%; float: left; margin-right: 10px; margin-bottom: 160px" /></tr></td>
</table>

Long read sequencing is revolutionary for evolutionary biology research. Generating high quality genomic resources is a reachable goal for non-model organisms. Our research utilizes state-of-the-art genomic sequencing technologies (in particular nanopore sequencing) to generate high-quality reference genome as a basis for studying the genome-wide variations of our plants of interest. We are also interested in using nanopore sequencing to develop and implement techniques that takes a deeper inference of the genomic variation. One such example is PoreC sequencing which is a long read sequencing based chromosome contact assay. Finally we are focused on developing bioinformatic methods to use the long read sequencing data or other genome-wide variation data to study the evolutionary basis of adaptation and speciation.

Related work: [Choi et al. 2022. Biorxiv](https://www.biorxiv.org/content/10.1101/2022.07.06.498888v1){:target="_blank"}; [Choi et al. 2021. PNAS](https://pubmed.ncbi.nlm.nih.gov/34497122/){:target="_blank"}; [Choi et al. 2020. Genome Biol](https://pubmed.ncbi.nlm.nih.gov/32019604/){:target="_blank"}
